### PR TITLE
docs: update info about time series

### PR DESCRIPTION
**Pull request by frodehk**
## Why is this pull request needed?

Difficult to find description of settings for TIME_SERIES TYPE: DEFAULT in eCalc Docs.

## What does this pull request change?

Improve description of TIME_SERIES keyword in general.

## Issues related to this change:
https://equinor-ecalc.atlassian.net/browse/ECALC-584?atlOrigin=eyJpIjoiZTg2ODE1YjJjOTE0NGRhZDk0MjRmNTU4NzE2NWU3MTMiLCJwIjoiaiJ9

### CODE REVIEW

File: docs/docs/about/references/keywords/TIME_SERIES.md [hunk 1]
```diff
@@ -16,4 +16,34 @@ later referred to in the system of energy consumers defined under [INSTALLATIONS
...
+
+| TYPE            | INTERPOLATION_TYPE                    | EXTRAPOLATION                          | INFLUENCE_TIME_VECTOR                                                                |
+|-----------------|---------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------|
+| DEFAULT         | Not allowed to specify. RIGHT is used | Not possible to specify. FALSE is used | Optional. Default: TRUE                                                              |
+| MISCELLANEOUS   | Required: LEFT, RIGHT or LINEAR       | Optional. Default: FALSE               | Optional. Default: TRUE                                                              |
+| **Description** | Determine if time steps should contribute to global time vector. TRUE or FALSE. At least one time vector is required to be TRUE.| Defines whether the rates in the source should be set to 0 after the last time step (FALSE), or equal to value at last time step after the time interval (TRUE). | Defines how rates are interpolated between the given time steps (LEFT/RIGHT/LINEAR). |
```
Comment (olelod) [inline docs/docs/about/references/keywords/TIME_SERIES.md:33]: Switch first and last column here.

Comment (olelod) [reply docs/docs/about/references/keywords/TIME_SERIES.md:33]: I mean under description. The description under INTERPOLATION_TYPE explains INFLUENCE_TIME_VECTOR and the other way around.

Comment (frodehk) [reply docs/docs/about/references/keywords/TIME_SERIES.md]: Nice catch! Done

Comment (frodehk) [reply docs/docs/about/references/keywords/TIME_SERIES.md]: Btw: is it too much to include description in the table for TIME_SERIES. The explanation of these keywords will also be under the corresponding keywords, e.g. INTERPOLATION_TYPE. Maybe delete the description row? Now it looks like this:

image

Comment (olelod) [reply docs/docs/about/references/keywords/TIME_SERIES.md]: I think it is a good idea to just remove the description here. And maybe also to flip the table, such that DEFAULT and MISCELLANEOUS are column headers, and each row corresponds to an attribute (INTERPOLATION_TYPE etc.)

Comment (frodehk) [reply docs/docs/about/references/keywords/TIME_SERIES.md]: Talked to @halsyl, and she wanted the description row. Keep it for now?

Comment (olelod) [reply docs/docs/about/references/keywords/TIME_SERIES.md]: Sure. But I think we should flip the table so it is in line with the required attributes above. One attribute per line.

Comment (frodehk) [reply docs/docs/about/references/keywords/TIME_SERIES.md]: Like this?
image

**Review by olelod — APPROVED** [review]

**Pull request merged**